### PR TITLE
fix: Improve projects responsiveness#4108

### LINKED ISSUE #4108: improve responsiveness of projects list page
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Checkout the video first

https://github.com/CircuitVerse/CircuitVerse/assets/57363826/7851bb4d-831a-48fc-a734-17afcdc6555e

The navbar and search bar exceed the device-width. It's not a good experience for end-user

### Steps To Reproduce

1. Visit this page [circuitverse.org/search?resource=Projects&q=jk&button=Submit](https://circuitverse.org/search?resource=Projects&q=jk&button=Submit) from mobile screen
2. Check the responsiveness issue

### Expected Behavior
The components should not exceed device width and page should not overflow and have any kind of scroll in x-axis

### Are you working on this issue?

No

### PR DESCRIPTION
**Pull request by coldrohak**
Fixes #4108

#### Describe the changes you have made in this PR -
Fixed the responsiveness of projects list page by containing the width of search-tags of projects and pagination overflow to scroll as in home page
### Screenshots of the changes (If any) -

image

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

**Comment by coldrohak** [comment]
Hey @tanmoysrt please review it

File: app/assets/stylesheets/components.scss [hunk 1]
```diff
@@ -328,6 +328,8 @@
   border-radius: 4px;
   color: $white;
   font-weight: 500;
+  max-width: 75%;
+  overflow: scroll;
```
Comment (tanmoysrt) [inline app/assets/stylesheets/components.scss:332]: In mobile version, the overflow is likely to be solved. But in desktop there is a unnecessary scroll bar coming.
image

I think, you can switch to `overflow: auto`, that may solve the issue

**Review by tanmoysrt — CHANGES_REQUESTED** [review]

**Comment by coldrohak** [comment]
@tanmoysrt yeah, now it doesn't 
circuitVersepr1

**Review by tanmoysrt — APPROVED** [review]
In mobile and desktop, the issues seems to be fixed !

cc @vedant-jain03

**Review by vedant-jain03 — APPROVED** [review]

**Comment by tachyons** [comment]
Screenshot 2023-12-05 at 1 17 20 AM
This broke the tags on profile, reverting

**Pull request merged**